scripts/brushless.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+source /opt/ros/noetic/setup.bash
+source /home/pi/catkin_ws/devel/setup.bash
+export PYTHONPATH=/home/pi/catkin_ws/devel/lib/python3/dist-packages:/opt/ros/noetic/lib/python3/dist-packages
+export ROS_IP=10.0.0.3
+roslaunch mrover brushless.launch
```

Comment:
This whole script file can be deleted now. It used to be the way to run the ros node on the pi with sudo, but now we have instructions to setup the .bashrc file to do all this sourcing on root shell startup.